_docs-sources/guides/production-framework/recipes/dev-team-experience.md

```diff
@@ -0,0 +1,93 @@
+    1. Creates a new repo called `search` in your company's GitHub org.
+    2. Generates the scaffolding for the Ruby on Rails app and commits the code to the `search` repo. The scaffolding includes all the basic libraries, packaging tools, monitoring code, automated test scaffolding, and everything else a web service needs to have to meet your company's requirements.
+    3. Configures a CI / CD pipeline for the `search` repo so that you get automatic builds and tests (but not yet deployment) for the app.
+    4. Configures required code reviews for every merge to the `main` branch.
+    5. Provides Ann with instructions on how to check out the repo and get started.
+
+## Run the app
+
+1. Ann follows the instructions to `git clone` the repo.
+2. Inside, she finds a `README` with further instructions.
+3. Following the `README`, she runs `docker-compose up` to fire up the app.
+4. Now she can code locally and use `localhost:3000` to test manually.
+5. The app scaffold includes tests, so Ann can use `rails test` to test automatically.
+
+## Scaffold new features in the app
+
+1. Occasionally, Ann will come back to the Service Catalog to look up how to accomplish common tasks in the Ruby on Rails app.
+2. Example #1: The `search` service may need to read a secret, such as a database password. Secrets management should be one of the core use cases in your Service Catalog, so when Ann looks it up, the Service Catalog generates some example code for her showing, for example, how to read secrets from HashiCorp Vault while the app is booting, and checks that example code directly into a branch of the `search` repo for Ann to build on top of.
+3. Example #2: While Ann is working on the `search` service, she realizes that the service should only be accessible to logged in users, so she browses the Service Catalog for how to handle auth. The Service Catalog generates and checks in some code that shows Ann how to make API calls to your company's `auth` service, how to extract data from the response, and how to redirect the user to a login page if they aren't logged in.
+
+## Deploy the infrastructure
+
+1. Meanwhile, Bob heads over to the Service Catalog to set up the infrastructure for the new app. First, he sees the clouds your company uses:
+    1. AWS
+    2. Azure
+2. Due to the way the ACLs are setup in your Service Catalog, the organization Ann and Bob are in only has access to AWS, so that's what Bob picks.
+3. Next, he sees a list of services available:
+    1. App deployment (EKS)
+    2. Database (RDS)
+    3. Cache (ElastiCache)
+    4. (... etc ...)
+4. Bob wants to deploy just their app for now, so he picks "App deployment (EKS)".
+5. The Service Catalog prompts him to enter some information, such as:
+    1. The name of the app.
+    2. The repo the app's code is in.
+    3. How many replicas to run.
+    4. How much CPU and memory the app needs.
+    5. What domain name to configure for the app.
+    6. Who the service owners are.
```

Comment:
This is vague. Is this something that's specific to backstage? If you want this to be more generic - consider elaborating that this could be the name of the business unit or team that's responsible for the app.